backend/app/notify/slack.go

```diff
@@ -86,7 +73,7 @@ func (t *Slack) SendVerification(_ context.Context, _ VerificationRequest) error
 }
 
 func (t *Slack) String() string {
-	return "Slack@" + t.channelID
+	return "slack: " + t.channelName
```

Comment:
I think there is some benefit in knowing the channel ID as well, I propose to have them both returned here.